helper/helper_functions.py

```diff
@@ -22,21 +28,23 @@ def santize_db_url(db_url):
 
     return db_url
 
-def get_db_session(db_url=DB_URL_DEFAULT):
 
-    sanitized_db_url = santize_db_url(db_url)
-    engine = create_engine(sanitized_db_url)
-    meta = MetaData()
-    meta.reflect(bind=engine)
+def init_engine():
+    try:
+        # initialize engine and session maker
+        engine = create_engine(santize_db_url(DATABASE_URL), poolclass=NullPool)
+        meta = MetaData()
+        meta.reflect(bind=engine)
+
+        return sessionmaker(bind=engine)
+    except:
+        return None
```

Comment:
Probably the wrong time to be asking this, but does anything actually check and handle the situation where this returns `None`?

Reply:
No, not really. I'm not sure how to handle this. I should at least report this in the log (I'm going to do this), but if, for example, the DB is down, can we automatically act?

Reply:
I tried to partially handle this [here](https://github.com/PerfectFit-project/virtual-coach-db/pull/80/commits/d6bd045c0e6d847b82c91d621ac2e7281bc0e455). I hope that this works. I should actually refactor a bit, but it would introduce a lot of madness.